pkg/templates/partitions.go

```diff
@@ -48,9 +48,16 @@ func (p *partitions) GetAgentPartitions(diskSize, recoveryIsoSize, dataIsoSize i
 	recoveryStartSector := recoveryEndSector - (recoveryPartitionSize / sectorSize)
 	recoveryStartSector = roundToNearestSector(recoveryStartSector, sectorAlignmentFactor)
 
+	// Calc root partition start/end sectors
```

Comment:
not a must: maybe do a helper function here. but really just a nit.

Reply:
sure, worth considering a refactor for a consecutive PR